D:\Program\Python\python.exe "D:\Math\PINNs\main\continuous_time_identification (Navier-Stokes)\demo.py" 
It: 0, Loss: 7.613e-01, l1: 0.001, l2: -0.00100
It: 100, Loss: 1.163e-01, l1: -0.003, l2: -0.00124
It: 200, Loss: 1.081e-01, l1: -0.013, l2: -0.00148
It: 300, Loss: 9.771e-02, l1: -0.014, l2: -0.00175
It: 400, Loss: 9.524e-02, l1: 0.028, l2: 0.00040
It: 500, Loss: 9.474e-02, l1: 0.024, l2: 0.00057
It: 600, Loss: 9.439e-02, l1: 0.038, l2: 0.00081
It: 700, Loss: 9.402e-02, l1: 0.048, l2: 0.00084
It: 800, Loss: 9.358e-02, l1: 0.035, l2: 0.00024
It: 900, Loss: 9.309e-02, l1: 0.007, l2: -0.00052
It: 1000, Loss: 9.236e-02, l1: 0.015, l2: -0.00021
It: 1100, Loss: 9.148e-02, l1: 0.062, l2: 0.00119
It: 1200, Loss: 9.081e-02, l1: 0.088, l2: 0.00215
It: 1300, Loss: 9.028e-02, l1: 0.094, l2: 0.00262
It: 1400, Loss: 8.991e-02, l1: 0.094, l2: 0.00295
It: 1500, Loss: 8.956e-02, l1: 0.097, l2: 0.00341
It: 1600, Loss: 8.891e-02, l1: 0.111, l2: 0.00443
It: 1700, Loss: 7.803e-02, l1: 0.469, l2: 0.01470
It: 1800, Loss: 6.832e-02, l1: 0.774, l2: 0.01927
It: 1900, Loss: 5.952e-02, l1: 0.818, l2: 0.02193

Process finished with exit code 0